src/components/calendar/AssociatedClass.test.js

```diff
@@ -68,11 +68,46 @@ describe('AssociatedClass', () => {
     },
   };
   const color = 'some color';
-  const associatedClass = { event, color, column: 0, columnWidth: 1 };
+  const associatedClass = {
```

Comment:
It might be good to make default test objects for each type, and then import and modify as needed.

Reply:
I was thinking that as well, I'll update the PR